karma.conf.js

```diff
@@ -16,6 +16,13 @@ module.exports = function(config) {
   config.set({
     // base path, that will be used to resolve files and exclude
     basePath: '',
+    plugins: [
+      'karma-coverage',
+      'karma-mocha',
+      'karma-ng-html2js-preprocessor',
+      'karma-sinon-chai',
+      'karma-chrome-launcher',
```

Comment:
I wanted to run karma from the command line without grunt making it hard to work with. I'm not sure why this isn't needed with the grunt setup but you need to declare these plugins to work with karma directly.